lib/express-rate-limit.js

```diff
@@ -71,7 +71,8 @@ function RateLimit(options) {
             limit: max,
             current: current,
             remaining: Math.max(max - current, 0),
-            resetTime: resetTime
+            resetTime: resetTime,
+            deltaSeconds: Math.ceil((resetTime.getTime() - Date.now()) / 1000)
```

Comment:
`resetTime` is not guaranteed to exist, so you need to check for it first

Reply:
Oops! Yes